docs/examples/Parameters/Legacy_Parameters.ipynb

```diff
@@ -0,0 +1,67 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Legacy parameter examples\n",
+    "\n",
+    "These are parameters which are no longer used in the modern implementations of QCoDeS but are kept here as examples for applications which still feature them.\n",
+    "\n",
+    "## ArrayParameter\n",
+    "> Note: This is an older base class for array-valued parameters. For any new driver we strongly recommend using `ParameterWithSetpoints` class which is both more flexible and significantly easier to use. Refer to notebook on [writing drivers with ParameterWithSetpoints](Simple-Example-of-ParameterWithSetpoints.ipynb). \n",
+    "\n",
+    "We have kept the documentation shown below of `ArrayParameter` for the legacy purpose.\n",
+    "For actions that create a whole array of values at once. When you use it in a `Loop`, it makes a single `DataArray` with the array returned by `get` nested inside extra dimension(s) for the loop. \n",
```

Comment:
I think we can remove this stuff about the loop

Reply:
agree